src/profiles_management/create.py

```diff
@@ -0,0 +1,89 @@
+"""Module responsible for creating and updating Profiles based on a PMR.
+
+This module includes both
+1. the high level functions for updating/creating Profiles based on a PM
+2. helpers to that will handle the different phases of the above logic
+
+The main function exposed is the create_or_update_profile(pmr), which
+will run an update on the whole cluster's Profiles anad Contributors
+based on a PMR.
+"""
+
+import logging
+from typing import Dict
+
+from lightkube.generic_resource import GenericGlobalResource
+
+from profiles_management.helpers.k8s import get_name
+from profiles_management.helpers.kfam import (
+    delete_contributor_authorization_policy,
+    delete_contributor_rolebinding,
+    list_contributor_authorization_policies,
+    list_contributor_rolebindings,
+)
+from profiles_management.helpers.profiles import list_profiles
+from profiles_management.pmr.classes import ProfilesManagementRepresentation
+
+log = logging.getLogger(__name__)
+
+
+def remove_access_in_stale_profile(profile: GenericGlobalResource):
```

Comment:
suggestion: I think this function name is more accurate, right?
```suggestion
def remove_access_to_stale_profile(profile: GenericGlobalResource):
```

Reply:
Great!